{
	"id": "CVE-2021-27379",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-27379",
				"description": "An issue was discovered in Xen through 4.11.x, allowing x86 Intel HVM guest OS users to achieve unintended read/write DMA access, and possibly cause a denial of service (host OS crash) or gain privileges. This occurs because a backport missed a flush, and thus IOMMU updates were not always correct. NOTE: this issue exists because of an incomplete fix for CVE-2020-15565.",
				"severity": [
					{
						"type": "cvss_v2",
						"source": "nvd@nist.gov",
						"cvss_v2": {
							"vector": "AV:L/AC:M/Au:N/C:P/I:P/A:C",
							"base_score": 5.9,
							"nvd_base_severity": "MEDIUM",
							"temporal_score": 5.9,
							"nvd_temporal_severity": "MEDIUM",
							"environmental_score": 5.9,
							"nvd_environmental_severity": "MEDIUM"
						}
					},
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 7.8,
							"base_severity": "HIGH",
							"temporal_score": 7.8,
							"temporal_severity": "HIGH",
							"environmental_score": 7.8,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"NVD-CWE-Other"
						]
					}
				],
				"references": [
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "http://www.openwall.com/lists/oss-security/2021/02/23/1"
					},
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "http://xenbits.xen.org/xsa/advisory-366.html"
					},
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "https://www.debian.org/security/2021/dsa-4888"
					},
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "https://xenbits.xen.org/xsa/advisory-366.html"
					},
					{
						"source": "cve@mitre.org",
						"url": "http://www.openwall.com/lists/oss-security/2021/02/23/1"
					},
					{
						"source": "cve@mitre.org",
						"url": "http://xenbits.xen.org/xsa/advisory-366.html"
					},
					{
						"source": "cve@mitre.org",
						"url": "https://www.debian.org/security/2021/dsa-4888"
					},
					{
						"source": "cve@mitre.org",
						"url": "https://xenbits.xen.org/xsa/advisory-366.html"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-27379"
					}
				],
				"published": "2021-02-18T17:15:15.867Z",
				"modified": "2024-11-21T05:57:52.597Z"
			},
			"segments": [
				{
					"ecosystem": "cpe"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "cpe",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"type": "cpe",
																"cpe": "cpe:2.3:o:debian:debian_linux:10.0:*:*:*:*:*:*:*"
															}
														}
													}
												]
											}
										]
									}
								]
							},
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"type": "cpe",
																"cpe": "cpe:2.3:o:xen:xen:*:*:*:*:*:*:x86:*"
															},
															"affected": {
																"type": "semver",
																"range": [
																	{
																		"lt": "4.12.0",
																		"ge": "3.2.0"
																	}
																]
															}
														}
													}
												]
											}
										]
									}
								]
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2021/CVE-2021-27379.json"
		]
	}
}
